web/views/backend/branch/index.php

```diff
@@ -0,0 +1,45 @@
+<div class="page-title">
+  <h4><?php echo $this->lang->line('user_list'); ?></h4>
```

Comment:
sửa echo $this->lang->line('user_list'); thành echo $this->lang->line(branch_list');